CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ### Bugfixes
 
-* Lorem ipsum.
+* Compact now throws an exception if writing fails for some reason
```

Comment:
I suggest adding that this would likely happen when running out of disk space.

Reply:
Updated